ModeratorFrontEnd/AIForOrcas/AIForOrcas.Server/Controllers/DetectionsController.cs

```diff
@@ -436,23 +448,25 @@ public async Task<IActionResult> GetFalsePositivesAsync([FromQuery] DetectionQue
 				// total number of records
 				double recordCount = queryable.Count();
 
+				var results = queryable
+					.Select(x => DetectionProcessors.ToDetection(x)).ToList();
```

Comment:
Is ToList necessary here? (see relevant comments in DetectionProcessors.cs below)